trieste/acquisition/sampler.py

```diff
@@ -81,14 +96,16 @@ def sample(self, model: ProbabilisticModel, sample_size: int, at: TensorType) ->
         :param sample_size: The desired number of samples.
         :param at: Where to sample the predictive distribution, with shape `[N, D]`, for points
             of dimension `D`.
+        :param select_output: A method that returns the desired output from the model sampler, with
+            shape `[S, N]` where `S` is the number of samples and `N` is the number of locations.
```

Comment:
maybe say that the default (and almost always to be used) option is to get the first sample?

Reply:
and for all the others :param select_output: doc bits